README.md

```diff
@@ -466,3 +466,13 @@ $ oc scale machineset ostest-worker-0 --replicas=3 -n openshift-machine-api
 machineset.machine.openshift.io/ostest-worker-0 scaled
 ```
 
+### Deploying dummy remote cluster nodes
+
+It is possible to add remote site nodes along with their own L2 network. To do so, use the 
+`remote-cluster.sh` script to create/teardown definitions of VMs and their corresponding network.
```

Comment:
Where's the `remote-cluster.sh` script?

Reply:
This is an error. I'll update the README. The name of the script had changed a few times.